views/post.ejs

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <% const title = ctx.prismicH.asText(post.data.title); %> 
+    <%- include('./partials/head', {title}); %>
+  </head>
+  <body>
+    <div class="main">
+      <div class="outer-container">
+        <div class="back">
+          <a href="./">back to list</a>
+        </div>
+        <h1><%- ctx.prismicH.asText(post.data.title) %></h1>
```

Comment:
You already have the title stored in a const. Reuse it here:
```
<h1><%- title %></h1>
```

Reply:
Done.